packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2988,21 +3009,6 @@ export class ContainerRuntime
 				forcedFullTree,
 			} as const;
 
-			// If validateSummaryBeforeUpload is true, validate that the summary generated by the summarizer nodes is
-			// correct before this summary is uploaded.
-			if (this.validateSummaryBeforeUpload) {
-				const validateResult = this.summarizerNode.validateSummary();
-				if (!validateResult.success) {
-					const { success, ...loggingProps } = validateResult;
-					const error = new RetriableSummaryError(
-						validateResult.reason,
-						validateResult.retryAfterSeconds,
-						{ ...loggingProps },
-					);
-					return { stage: "base", ...generateSummaryData, error };
```

Comment:
I understand what you're saying about the generated summary data not being trustworthy so don't log it.  If you think there could be a case where logging the _incorrect_ info could help debug/diagnose a bug, then maybe we leave it but put it in a details-type object?

Reply:
With the current state of things, this generate data here is not logged or used at all. SummaryGenerator filters it out because the stage is base.
I tried adding support to log these extra params in details but its too much plumbing to make it worth and I don't believe its worth it. The extra params are not likely to help with any debugging.